packages/eslint-config-hop-protocol/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "eslint-config-hop-protocol",
+  "version": "0.0.1-beta.2",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "version:sync": "package-sync-version",
+    "release": "npm run publish:latest",
+    "version:bump": "npm version prerelease --preid=beta",
+    "bump": "npm run version:bump && npm run version:sync",
+    "bump-release": "npm run version:bump && npm run version:sync && npm run release"
+  },
+  "author": "",
+  "license": "MIT",
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^6.18.1",
+    "@typescript-eslint/parser": "^6.18.0",
+    "eslint": "^8.56.0",
+    "eslint-config-standard": "^16.0.2",
+    "eslint-plugin-jest": "^27.6.1",
+    "eslint-plugin-json": "^3.1.0",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-node": "^11.1.0",
```

Comment:
I believe `eslint-plugin-node` is [unmaintained](https://github.com/mysticatea/eslint-plugin-node/issues/300) and has been replaced with [`eslint-plugin-n`](https://github.com/eslint-community/eslint-plugin-n).

> forked from [eslint-plugin-node](https://www.npmjs.com/package/eslint-plugin-node) v11.1.0. as the original repository seems https://github.com/mysticatea/eslint-plugin-node/issues/300.

In that case, we can use `eslint-plugin-n`.

Reply:
Resolved with https://github.com/hop-protocol/hop/pull/545/commits/19782a3ed6c46210c7cd1fea8dc39b26806c93d9